src/router_core/modules/mobile_sync/mobile.c

```diff
@@ -51,6 +51,8 @@ static const char *HAVE_SEQ   = "have_seq";
 #define ADDR_SYNC_TO_BE_DELETED         0x00000004
 #define ADDR_SYNC_MOBILE_TRACKING       0x00000008
 
+#define ADDR_SYNC_MA_REQUESTED          0x00000001
```

Comment:
This leads me to believe this is a flag for Address.sync_mask - possible alias for IN_ADD_LIST.   Better name and comment requested.